### ISSUE TITLE
Is this project still active as it's not been updated for some time?

### ISSUE THREAD

**Issue by jasonsedwards** [hashicorp/consul-template#2009]
Is this project still active, as it has not been updated for some time?
There are several CVEs within it that have been patched in unmerged PRs

**Comment by blake** [comment]
@jasonsedwards Yes, the project is still active and there have been recent commits to the repository.

**Comment by noorul** [comment]
It will be great to see some roadmaps. Also I see that CVEs are found by various scanners.

**Comment by blake** [comment]
@noorul We will be addressing CVEs in this repository and evaluating feature requests or bug reports.

**Comment by noorul** [comment]
Any timelines for this?

**Comment by noorul** [comment]
@blake Any timelines for the next release?

**Comment by jasonsedwards** [comment]
It'd be useful to know when you intend to release a new version as there are several CVEs within your last release that are being identified during security scans.

**Comment by oyelekci** [comment]
Is there an alternative to this tool? We're now unable to continue using it in production due to unfixed vulnerabilities.

**Comment by axi92** [comment]
Same here, there are many unfixed CVEs. Would be nice to see those fixed soon with a release.

**Comment by nickwales** [comment]
All vulnerabilities up to Feb 24th 2025 have been patched in the v0.40.0 release. There are new vulnerabilities that will be patched in upcoming releases.
